Assets/Scripts/Mecha/Gunner/CannonBehavior.cs

```diff
@@ -14,26 +14,34 @@ public class CannonBehavior : MonoBehaviour {
     public int LaserEnergyConsumption = 1;
     public int MissileEnergyConsumption = 10;
 
+    public float aimRange = Mathf.Infinity;
+    public Camera cam;
+
     private MechaController mechaController;
 
     private float nextFire;
 
+    private RaycastHit hitInfo;
+    private GameObject projectile;
```

Comment:
`projectile` being used in only one method, it's not useful to declare it as a class field